ui/components/component-library/banner-base/banner-base.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import classnames from 'classnames';
+
+import {
+  BackgroundColor,
+  BorderRadius,
+  Display,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+
+import {
+  ButtonLink,
+  IconName,
+  ButtonIcon,
+  Text,
+  Box,
+  ButtonLinkSize,
+  ButtonIconSize,
+} from '..';
+import { BoxProps, PolymorphicRef } from '../box';
+import { BannerBaseComponent, BannerBaseProps } from './banner-base.types';
+
+export const BannerBase: BannerBaseComponent = React.forwardRef(
+  <C extends React.ElementType = 'div'>(
```

Comment:
trying to find a fix for this, input would be appreciated
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/199cb66e-89ec-481a-bfe5-795bd065ba8d)